libs/product/src/models/composite-product-item.ts

```diff
@@ -0,0 +1,17 @@
+import { DaffProduct } from './product';
+
+export interface DaffCompositeProductItem {
+	id: number;
+	required: boolean;
+	title: string;
+	type: string;
```

Comment:
What does this `type` refer to?

Reply:
I changed the name to `input_type`. It is the type of input used to select the composite product option. Like "select", "radio", etc